arm/config/config.py

```diff
@@ -15,15 +18,55 @@ def _load_config(fp):
     return config
 
 
-def load_abcde(fp):
+def _load_abcde(fp):
     with open(fp, "r") as abcde_read_file:
         config = abcde_read_file.read()
     return config
 
 
-# aprise config, open and read yaml contents
-apprise_config = _load_config(apprise_config_path)
 # arm config, open and read yaml contents
+# handle arm.yaml migration here
+# 1. Load both current and template arm.yaml
+cur_cfg = _load_config(arm_config_path)
+new_cfg = _load_config("/opt/arm/setup/arm.yaml")
+
+# 2. If the dicts do not have the same number of keys
+if len(cur_cfg) != len(new_cfg):
+    # 3. Update new dict with current values
+    for key in cur_cfg:
+        if key in new_cfg:
+            new_cfg[key] = cur_cfg[key]
```

Comment:
Somewhere around here it converts all of the `true` to 1 and 'false' to 0 in the arm.yaml and then saves it

This isnt really a huge issue if they are passed as ints but if they are treated as strings then they are all true. Also it may be confusing for some users.
Will need fixed to show `true` or `false` again before this can be merged.

Reply:
See my comment below